src/adapters/axelar/AxelarSenderAdapter.sol

```diff
@@ -80,20 +59,20 @@ contract AxelarSenderAdapter is IBridgeSenderAdapter, BaseSenderAdapter {
 
         string memory destinationChain = chainIdMap[_toChainId];
 
-        if (bytes(destinationChain).length > 0) {
+        if (bytes(destinationChain).length <= 0) {
             revert Error.INVALID_DST_CHAIN();
         }
 
         msgId = _getNewMessageId(_toChainId, _to);
-        _callContract(destinationChain, StringAddressConversion.toString(receiverAdapter), msgId, _to, _data);
+        _callContract(destinationChain, receiverAdapter, msgId, _to, _data);
 
         emit MessageDispatched(msgId, msg.sender, _toChainId, _to, _data);
     }
 
     /// @dev maps the MMA chain id to bridge specific chain id
     /// @dev _origIds is the chain's native chain id
     /// @dev _axlIds are the bridge allocated chain id
-    function setChainchainIdMap(uint256[] calldata _origIds, string[] calldata _axlIds) external onlyCaller {
+    function setChainIdMap(uint256[] calldata _origIds, string[] calldata _axlIds) external onlyCaller {
```

Comment:
have we checked to see if Axelar provides any deployed registry contract that offers such a mapping already, that we can call? I fear that maintaining this mapping might prove problematic for us over time since each update has to go through a governance vote.